lambda-buffers-compiler/src/LambdaBuffers/Logger.hs

```diff
@@ -0,0 +1,41 @@
+{- | Module: LambdaBuffers.Logger
+
+Utilities for logging INFO, and ERROR messages.
+
+This follows the GNU error message format:
+<https://www.gnu.org/prep/standards/html_node/Errors.html>
+-}
+module LambdaBuffers.Logger (logInfo, logError, logErrorWithSourceSpan) where
+
+import System.IO qualified
+
+{- | Logs an INFO message of the format
+> <file>: info: <msg>
+if @<file>@ is non empty, otherwise, it prints
+> info: <msg>
+-}
+logInfo :: FilePath -> String -> IO ()
+logInfo "" msg = System.IO.hPutStrLn System.IO.stderr $ "info: " <> msg
```

Comment:
Shouldn't this be `System.IO.stdout` for an info log?

Reply:
Hmm not sure what I was thinking here, but I vaguely remember using stderr for any logging for some reason